cmd/incusd/instance_access.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/gorilla/mux"
+
+	internalInstance "github.com/lxc/incus/v6/internal/instance"
+	"github.com/lxc/incus/v6/internal/server/request"
+	"github.com/lxc/incus/v6/internal/server/response"
+)
+
+// swagger:operation GET /1.0/instances/{name}/access
+//
+//	Get who has access to an instnace
+//
+//	---
+//	produces:
+//	  - application/json
+//	parameters:
+//	  - in: query
+//	    name: project
+//	    description: Project name
+//	    type: string
+//	responses:
+//	  "200":
+//	    description: Access
+//	    schema:
+//	      type: list
+//	      description: Sync response
+//	      properties:
+//	        type:
+//	          type: object
+//	          description: info for each access entry
+//	          properties:
+//		        identifier:
+//		          type: string
+//		          description: certificate fingerprint
+//		          example: 636b69519d27ae3b0e398cb7928043846ce1e3842f0ca7a589993dd913ab8cc9
+//		        role:
+//		          type: string
+//		          description: the role associated with the certificate
+//		          example: admin
+//		        provider:
+//		          type: string
+//		          description: Which authorization method the certificate uses
+//		          example: tls
+//	  "400":
+//	    $ref: "#/responses/BadRequest"
+//	  "403":
+//	    $ref: "#/responses/Forbidden"
+//	  "500":
+//	    $ref: "#/responses/InternalServerError"
+func instanceAccess(d *Daemon, r *http.Request) response.Response {
+	s := d.State()
+
+	instanceType, err := urlInstanceTypeDetect(r)
+	if err != nil {
+		return response.SmartError(err)
+	}
+	
+	projectName := request.ProjectParam(r)
+	name, err := url.PathUnescape(mux.Vars(r)["name"])
+	if err != nil {
+		return response.SmartError(err)
+	}
+
+	if internalInstance.IsSnapshot(name) {
+		return response.BadRequest(fmt.Errorf("Invalid instance name"))
+	}
+
+	// Handle requests targeted to a container on a different node
+	resp, err := forwardedResponseIfInstanceIsRemote(s, r, projectName, name, instanceType)
+	if err != nil {
+		return response.SmartError(err)
+	}
+
+	if resp != nil {
+		return resp
+	}
+
```

Comment:
This isn't needed, we don't need to run the logic on the machine that's running the instance.